src/cart/cart.controller.ts

```diff
@@ -11,35 +23,96 @@ import { CartService } from './services';
 export class CartController {
   constructor(
     private cartService: CartService,
-    private orderService: OrderService
-  ) { }
+    private orderService: OrderService,
+  ) {}
 
   // @UseGuards(JwtAuthGuard)
   // @UseGuards(BasicAuthGuard)
-  @Get()
-  findUserCart(@Req() req: AppRequest) {
-    const cart = this.cartService.findOrCreateByUserId(getUserIdFromRequest(req));
+  @Get(':cartid')
+  async findUserCart(@Req() req: AppRequest) {
+    try {
+      console.log('req & params!!!: ', req.params, req.params.cartid);
+      const cart: QueryResult = await client(
```

Comment:
Please reconsider your approach to commands creation. Currently your code is vulnerable to SQL Injection attack.
Please use parameterized commands:
https://node-postgres.com/features/queries#parameterized-query